### PR TITLE
Enforce google-style-format and added checkstyle check for CI testing. 

### PR DESCRIPTION
**Pull request by kurtisvg**

### CODE REVIEW

File: connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
 import jnr.unixsocket.UnixSocketAddress;
 import jnr.unixsocket.UnixSocketChannel;
 
+
```
Comment (netdpb) [inline connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java]: Only one blank line needed here.

File: connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java [hunk 2]
```diff
@@ -84,7 +85,9 @@ public Socket afterHandshake() {
     return socket;
   }
 
-  /** Returns {@code true} if running in a Google App Engine Standard runtime. */
+  /**
```
Comment (netdpb) [inline connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java]: Hmm. google-java-format usually prefers single-line Javadoc if it fits. Are you running that?

File: core/src/main/java/com/google/cloud/sql/CredentialFactory.java [hunk 3]
```diff
@@ -20,7 +20,7 @@
 
 /**
  * Factory for creating {@link Credential}s for interaction with Cloud SQL Admin API.
- */
+ * */
```
Comment (netdpb) [inline core/src/main/java/com/google/cloud/sql/CredentialFactory.java]:
```suggestion
 */
```

google-java-format should have caught this.

File: connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java (hunk 2 above)
Comment (netdpb) [reply connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java]: OK. I don't personally use Checkstyle, even with Google settings. What value do you find it has?

File: pom.xml [hunk 4]
```diff
@@ -1,183 +1,216 @@
...
+    </snapshotRepository>
+
+    <repository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+  </distributionManagement>
+
+  <scm>
+    <connection>scm:git:https://github.com/GoogleCloudPlatform/cloud-sql-mysql-socket-factory.git
```
Comment (netdpb) [inline pom.xml]: This looks odd.

File: connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java (hunk 2 above)
Comment (kurtisvg) [reply connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java]: Checkstyle is enforcement of the style. I spent some time experimenting with using google-java-format and diffing, but checkstyle provides a better explanation of why something is incorrect and is more accessible to Windows users.

**Pull request merged**